app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
@jrafanie @Fryguy @gtanzillo 

this works, I found it here https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request_workflow.rb#L8

is this ok ?

thanks!